deployment.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
You should use the new KafkaTopic from master. And I think you can also add the KafkaUser resource for the USer Operator.